qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/search/SearchCustomerOutput.groovy

```diff
@@ -19,5 +19,6 @@ interface SearchCustomerOutput extends UseCaseFailure {
    *
    * @param foundCustomers A list of {@link Customer}.
    */
-  void successNotification(List<Customer> foundCustomers)
-}
+
+  void reportFoundCustomers(List<Customer> customers)
```

Comment:
I would never change the name of the interface, there is no such change reflected in our diagram. Why the change?